### DISCUSSION TITLE
About ISSUE TEMPLATES

### DISCUSSION THREAD

**Discussion by Kshitiz-Mhto** [fonoster/yaps#420] [Q&A] [answered]
hi @psanders i got know about this project from twitter and i want to contribute to this project

is there any plan regarding upgrading the issue templates from `md` to `yml` ?
i want to upgrade the issue template,  will it do some good to the repo?

**Comment by psanders** [comment]
@Kshitiz-Mhto

What are the pros/cons of switching?

cc @YuriCodes

**Comment by YuriCodes** [comment, marked as answer]
Hello! Thank you for your interest in contributing, we are all about community!

Pros: it would look the same
Cons: it would disrupt the flow of the GitHub if we start adding yml files even if it shows up the same in browser. Also, there's more versatility with markdown syntax, it is also more intuitive, widely adopted and easy formatting specially for longer documents.

If you have any more questions or suggestions we encourage you to join our [Discord community](https://discord.gg/mpWSRUhG7e) where you can connect with other contributors, maintainers and stay up to date with what we have in store for the upcoming months.
